### ISSUE TITLE
[UGQL] Mutation for a required, multiple field doesn't allow working with multiple values + workaround

### ISSUE THREAD

**Issue by ZubatyNos** [matfyz-sk/courses-backend#40]
Having this parameter in a Payload class, doesn't produce expected behavior.
```js
 hBrother: {
      required: true,
      multiple: true,
      dataType: 'node',
      objectClass: "payload"
},
```

Query schema is correct
![image](https://github.com/matfyz-sk/courses-backend/assets/59411496/2c48a19a-7478-4ef3-83d3-fe6a85868ce0)

Mutation schema is not
![image](https://github.com/matfyz-sk/courses-backend/assets/59411496/cf613d0f-31ad-4b74-af3a-c0520e4e24fe)

The fix is making the field non-required.
```js
 hBrother: {
      required: **false**,
      multiple: true,
      dataType: 'node',
      objectClass: "payload"
},
```

**Comment by crnkjck** [comment]
@bl96 has probably fixed this in https://github.com/matfyz-sk/UltraGraphQL/commit/dd637d5523da6810feee47f870d5727a163ff902.
@ZubatyNos, could you bump the version of UGQL, build it, and commit the `jar` here?

**Comment by ZubatyNos** [comment]
Fixed as of UGQL v1.4.10 in commit https://github.com/matfyz-sk/UltraGraphQL/commit/dd637d5523da6810feee47f870d5727a163ff902
